tvb/simulator/models/epileptorcodim3.py

```diff
@@ -0,0 +1,595 @@
+# -*- coding: utf-8 -*-
+#
+#
+#  TheVirtualBrain-Scientific Package. This package holds all simulators, and
+# analysers necessary to run brain-simulations. You can use it stand alone or
+# in conjunction with TheVirtualBrain-Framework Package. See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2017, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+
+"""
+Saggio codimension 3 Epileptor model
+
+.. moduleauthor:: Len Spek
+
+"""
+
+from .base import Model, LOG, numpy, basic, arrays
+
+
+class EpileptorCodim3(Model):
+    r"""
+    .. [Saggioetal_2017] Saggio ML, Spiegler A, Bernard C, Jirsa VK. *Fast–Slow Bursters in the Unfolding of a
+        High Codimension Singularity and the Ultra-slow Transitions of Classes.*
+        Journal of Mathematical Neuroscience. 2017;7:7. doi:10.1186/s13408-017-0050-8.
+
+    .. The Epileptor codim 3 model is a neural mass model which contains two subsystems acting at different timescales.
```

Comment:
please reflow to 80 columns, in Vim this is `gqip` but your editor may be different. or I can do it for you.

Reply:
I will adjust this